scripts/newinstall.sh

```diff
@@ -165,48 +180,57 @@ if true; then
 fi
 
 
-##########	Test/warn about Python versions, offer to get miniconda2 if too old
-##########	LSST currently mandates Python 2.7 and no other.
+##########	Test/warn about Python versions, offer to get miniconda if not supported.
+##########	LSST currently mandates Python 2.7 but 3.5 is arriving.
 ##########	We assume that the python in PATH is the python that will be used to
 ##########	build the stack if miniconda2 is not installed.
 
 if true; then
-	PYVEROK=$(python -c 'import sys; print("%i" % (sys.hexversion >= 0x02070000 and sys.hexversion < 0x03000000))')
+	# Check the version by running a small Python program (taken from the Python EUPS package)
+	PYVEROK=$(python -c 'import sys
+minver2=7
+minver3=5
+vmaj = sys.version_info.major
+vmin = sys.version_info.minor
+if (vmaj == 2 and vmin >= minver2) or (vmaj == 3 and vmin >= minver3):
+    print(1)
+else:
+    print(0)')
 	if [[ "$batch_flag" = true ]]; then
-		WITH_MINICONDA2=1
+		WITH_MINICONDA=1
 	else
 		if [[ $PYVEROK != 1 ]]; then
 			cat <<-EOF
 
-			LSST stack requires Python 2.7; you seem to have $(python -V 2>&1) on your
+			LSST stack requires Python 2.7 or >=3.5; you seem to have $(python -V 2>&1) on your
```

Comment:
If we're going to allow a (nonexistent) 2.8 in the code, shouldn't you put a ">=" on the 2.7, too?